test/google_api_headers_test.mocks.dart

```diff
@@ -0,0 +1,27 @@
+// Mocks generated by Mockito 5.0.0-nullsafety.7 from annotations
```

Comment:
I prefer to not check in auto-generated files into git, can you do the followings: 
- [ ] Update `.gitignore` to ignore `*.mocks.dart` files
- [ ] Update `.github/workflows/analyze_test.yml` to include a step to generate this file

If you're not sure about anything, let me know and I can help you out

Reply:
I wil ignore it